Sources/Metrics/Metrics.swift

```diff
@@ -143,8 +146,9 @@ extension Timer {
     public func measure<Result, Failure: Error, Clock: _Concurrency.Clock>(
         clock: Clock = .continuous,
         isolation: isolated (any Actor)? = #isolation,
-        body: () async throws(Failure) -> sending Result
-    ) async throws(Failure) -> sending Result where Clock.Duration == Duration {
+        // DO NOT FIX THE WHITESPACE IN THE NEXT LINE UNTIL 5.10 IS UNSUPPORTED
+        // https://github.com/swiftlang/swift/issues/79285
+        body: () async throws(Failure) -> sending Result ) async throws(Failure) -> sending Result where Clock.Duration == Duration {
```

Comment:
As currently written, this doesn't seem to be sufficient to bypass the assertion. I suggest:
```suggestion
        body: () async throws(Failure) -> sending Result) async throws(Failure) -> sending Result where Clock.Duration == Duration {
```